### DISCUSSION TITLE
ios运行白屏问题

### DISCUSSION THREAD

**Discussion by 1243excz** [galacean/discussion#120] [Editor] [answered]
我在编辑器里面下载项目（界面上只放了两个精灵图），本地运行。ios 12，版本：15.4.1会出现退出再进白屏问题。

https://github.com/galacean/discussion/assets/167043713/0cb893e2-70f0-49f6-a203-3a286ead3b80

**Comment by luzhuang** [comment, marked as answer]
是KTX2在 ios16版本以下的兼容问题： https://galacean.antgroup.com/engine/docs/latest/cn/graphics-texture-compression

**Reply by BUPTlhuanyu** [reply]
所以这个问题已经修复了？还是说 ios16 以下都要降级？或者不用 KTX2？

**Reply by eyworldwide** [reply]
@BUPTlhuanyu  文档里有说明如何降级： https://galacean.antgroup.com/engine/docs/graphics/texture/compression/#%E5%85%BC%E5%AE%B9%E6%80%A7
